manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -4,6 +4,8 @@
 apiVersion: image.openshift.io/v1
 kind: ImageStream
 metadata:
+  labels:
+    opendatahub.io/notebook-image: "false"
```

Comment:
There should be some comment before this line, to explain, I think.

Besides that,
/lgtm

Reply:
Thanks for the review, I added the comment